ci/bash-lib.yml

```diff
@@ -100,3 +100,4 @@ steps:
     }
     END
     echo "##vso[task.setvariable variable=${{parameters.var_name}}]$TMP"
+  displayName: install Bash lib
```

Comment:
Not technically related, but in looking at the job result for the past few days I noticed this job is reported as "Bash" in the Azure UI and that makes it harder to understand.